model/clusters_mgmt/v1/sts_type.model

```diff
@@ -27,4 +27,10 @@ struct STS {
 
 	// List of roles necessary to access the AWS resources of the various operators used during installation
 	OperatorIAMRoles []OperatorIAMRole
+
+    //Custom IAM roles
```

Comment:
I would change the comment here to:
* Custom IAM roles to use for the instance profiles of the master and worker instances